### ISSUE TITLE
dataload_format_to_csv script errors

### ISSUE THREAD

**Issue by bowenyou** [SymposiumOrganization/NeuralSymbolicRegressionThatScales#4]
Thanks for making the repo public! I'm trying to use the dataload_format_to_csv script within the scripts directory as in your README. I've followed the instructions exactly so far and when I am running 
`python3 scripts/csv_handling/dataload_format_to_csv.py raw_test_path=data/raw_datasets/150`

I get the following error:
```
scripts/csv_handling/dataload_format_to_csv.py:50: UserWarning: 
config_path is not specified in @hydra.main().
See https://hydra.cc/docs/next/upgrades/1.0_to_1.1/changes_to_hydra_main_config_path for more information.
  @hydra.main(config_name="../config")
Could not override 'raw_test_path'.
To append to your config use +raw_test_path=data/raw_datasets/150
Key 'raw_test_path' is not in struct
    full_key: raw_test_path
    object_type=dict

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
```

I haven't changed anything within the repo, and am just running it as is. Is there a fix for this?
Is it possible to adapt this script to not just the validation set but the training data as well?

Thanks!

**Comment by bowenyou** [comment]
think it has to do with version of hydra-core. Downgrading to 1.0.0 from 1.1.0 seems to fix this.

**Comment by TommasoBendinelli** [comment]
Hi bowenyou! Thanks for finding this out. I guess hydra 1.0.0 is needed because namespaces are handled differently.
